visual-testing-app/webpack.config.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-console */
+const path = require('path');
+
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+
+module.exports = {
+  target: 'web',
+  // note: we have to use 'development' for now, because when we use production
+  // our emotion styles aren't displayed in our percy snapshots.
+  mode: 'development',
+  stats: 'minimal',
+  entry: './src/index.js',
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist'),
+    publicPath: '/',
+  },
+  optimization: {
+    splitChunks: {
+      cacheGroups: {
+        vendors: {
+          test: /[\\/]node_modules[\\/]/,
+          name: 'vendors',
+          chunks: 'all',
+          priority: -20,
+        },
+        'application-components': {
+          test: /application-components/,
+          name: 'application-components',
+          chunks: 'all',
+          priority: -15,
+        },
+      },
+    },
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: [/(node_modules)/, /(application-components.es)/],
+        use: {
+          loader: 'babel-loader',
+          query: {
+            compact: true,
+            presets: [
+              require.resolve('@commercetools-frontend/babel-preset-mc-app'),
+            ],
+          },
+        },
+      },
+      {
+        test: /\.svg$/,
+        use: {
+          loader: require.resolve('svg-url-loader'),
```

Comment:
we need this because `PageNotFound` is doing 

```
import PageNotFoundSVG from '@commercetools-frontend/assets/images/page-not-found.svg';
```

So the file isn't actually base64 encoded yet.

Reply:
Can we use svgr here as we use that elsewhere too?

Reply:
Nah, we use SVGR just for SVGs ending in `.react.svg`. Generally icons.